### PR TITLE
Package coq-haskell.1.1.0

### PR DESCRIPTION
**Pull request by jwiegley**

### CODE REVIEW

File: released/packages/coq-haskell/coq-haskell.1.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+synopsis: "A library to provide Haskell-familiar constructions in Coq"
+description: """
+This library is designed for Haskell users who are either using Coq to build
+code intended for extraction to Haskell, or who wish to prototype/prove their
+algorithms in Coq. It provides a collection of definitions and notations to
+make Gallina more familiar to Haskellers.
+"""
+
+build: [make "JOBS=%{jobs}%" ]
```
Comment (palmskog) [inline released/packages/coq-haskell/coq-haskell.1.1.0/opam:18]: I remember we introduced a regular delegating makefile in the end, so this standard boilerplate should work:
```suggestion
build: [make "-j%{jobs}%"]
```

File: released/packages/coq-haskell/coq-haskell.1.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+version: "1.1.0"
```
Comment (palmskog) [inline released/packages/coq-haskell/coq-haskell.1.1.0/opam:3]:
```suggestion
```
This is already part of the file path, so not needed in the `opam` file itself. Makes it easier to copy opam packages.

File: released/packages/coq-haskell/coq-haskell.1.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+authors: [
+  "John Wiegley"
+]
+
+tags: [
+  "keyword:haskell"
+  "category:Miscellaneous/Coq Extensions"
+  "date:2022-03-29"
```
Comment (palmskog) [inline released/packages/coq-haskell/coq-haskell.1.1.0/opam:36]:
```suggestion
  "date:2022-07-22"
```

File: released/packages/coq-haskell/coq-haskell.1.1.0/opam [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+This library is designed for Haskell users who are either using Coq to build
+code intended for extraction to Haskell, or who wish to prototype/prove their
+algorithms in Coq. It provides a collection of definitions and notations to
+make Gallina more familiar to Haskellers.
+"""
+
+build: [make "JOBS=%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.14" & < "8.16~") | (= "dev")}
```
Comment (palmskog) [inline released/packages/coq-haskell/coq-haskell.1.1.0/opam:21]:
```suggestion
  "coq" {(>= "8.14" & < "8.17~") | (= "dev")}
```
The package works well on Coq 8.16+rc1, so I think we can say it's safe for the yet-to-be-released Coq 8.16.0

**Comment by jwiegley** [comment]
Thank you for the assist, @palmskog!

**Pull request merged**